test-suite/src/function/format.rs

```diff
@@ -0,0 +1,16 @@
+use crate::*;
+
+test_case!(format, async move {
+    use gluesql_core::{
+        executor::{EvaluateError, SelectError},
+        prelude::Value::*,
+    };
+
+    let test_cases = vec![(
+        r#"SELECT FORMAT(DATE "2017-06-15", "%Y-%m")"#,
+        Ok(select!("2017-06")),
```

Comment:
The column name and type are missing from the `select!` macro.

https://github.com/gluesql/gluesql/blob/7fadbf557505c35569fbcedb2e9c06604ea74ac6/test-suite/src/series.rs#L131-L138

Reply:
Thank you for pointing this out, I will apply this to the new test cases